src/main/webapp/resources/css/structure.css

```diff
@@ -129,6 +129,10 @@ a.tooltip-icon:hover, a.tool-tip:focus, a.tool-tip:active {text-decoration: none
 /* Dropdown submenu */
 .dropdown-submenu {
     position: relative;
+    z-index: 1000 !important;
+}
+#manageTemplatesForm\:allTemplates .ui-datatable-tablewrapper {
```

Comment:
From my testing, this was the only change I needed. Is zindex for the drop down needed for some case as well? Same for the style change and adding the bootstrap-button-tooltip - are the needed or just left-overs from testing?

Reply:
They were remnants of prior attempts at a fix. I have removed them. Thanks for the close eye.